idea/src/org/jetbrains/kotlin/idea/codeInsight/surroundWith/MoveDeclarationsOutHelper.kt

```diff
@@ -83,6 +83,35 @@ object MoveDeclarationsOutHelper {
         return PsiUtilCore.toPsiElementArray(resultStatements)
     }
 
+    private fun addSymbolAfterDeclaration(container: PsiElement, declaration: KtProperty, isLastStatemnt: Boolean) {
+        if (isLastStatemnt) {
+            container.addAfter(KtPsiFactory(declaration).createEQ(), declaration)
+        }
+        else {
+            container.addAfter(KtPsiFactory(declaration).createNewLine(), declaration)
```

Comment:
It should never be necessary to create newlines from text. All newlines should be inserted by the formatter.